components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -179,51 +179,86 @@ func runShowCmd(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		sshUser := infra.Outputs.SSHUser.Value
-		sskKeyFile := infra.Outputs.SSHKeyFile.Value
-		sshPort := infra.Outputs.SSHPort.Value
+		var outFrontend string
+		var outBackend string
+		var outFiles []string
+		if len(args) > 0 && args[0] != "" {
+			outFrontend = args[0]
+			outBackend = "> " + args[0]
+			outFiles = []string{outFrontend}
+		}
+		frontendCommand := fmt.Sprintf(GET_FRONTEND_CONFIG, outFrontend)
+		postgresqlCommand := fmt.Sprintf(GET_BACKEND_CONFIG, "postgresql", outBackend)
+		opensearchCommand := fmt.Sprintf(GET_BACKEND_CONFIG, "opensearch", outBackend)
+
+		frontend := &Cmd{
+			CmdInputs: &CmdInputs{
+				NodeType: false,
+			},
+		}
 
-		sshConfig := &SSHConfig{
-			sshUser:    sshUser,
-			sshKeyFile: sskKeyFile,
-			sshPort:    sshPort,
+		automate := &Cmd{
+			CmdInputs: &CmdInputs{
+				Cmd:         frontendCommand,
+				Single:      false,
+				InputFiles:  []string{},
+				Outputfiles: outFiles,
+				NodeType:    configCmdFlags.automate,
+			},
 		}
-		sshUtil := NewSSHUtil(sshConfig)
 
-		if configCmdFlags.overwriteFile {
-			writer.Errorln("Overwrite flag is not supported in HA")
+		chef_server := &Cmd{
```

Comment:
Rename this `chef_server` to camel case.

Reply:
Done